docs/rules/prefer-use-deep-compare-memo.md

```
@@ -0,0 +1,79 @@
+### When Not To Use It
+
+- Performance hotspots where deep comparison overhead is undesirable. You can disable the rule for a specific line:
+
+```ts
+// eslint-disable-next-line @blumintinc/blumint/prefer-use-deep-compare-memo
+const x = useMemo(() => compute(data), [data]);
+```
+
+## Version
+
+- Introduced in v1.10.0
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)